repl/src/test/scala/org/apache/spark/repl/ExecutorClassLoaderSuite.scala

```diff
@@ -118,8 +115,17 @@ class ExecutorClassLoaderSuite
     val resourceName: String = parentResourceNames.head
     val is = classLoader.getResourceAsStream(resourceName)
     assert(is != null, s"Resource $resourceName not found")
-    val content = Source.fromInputStream(is, "UTF-8").getLines().next()
-    assert(content.contains("resource"), "File doesn't contain 'resource'")
+
+    var bufferedSource: BufferedSource = null
+    Utils.tryWithSafeFinally {
+      bufferedSource = Source.fromInputStream(is, "UTF-8")
```

Comment:
Can this line be outside the try block? if it fails, there's nothing to clean up. Then you don't need a var